### ISSUE TITLE
Busca do Leilão não funciona

### ISSUE THREAD

**Issue by AnibalDuarte** [danielbarion/JBlueHeart-Source#14]
O Leilão no Community Board funciona bem, mas o campo de busca não executa a busca.

**Comment by danielbarion** [comment]
@LuizHFallen da uma olhada quando puder;

**Comment by LuizHFallen** [comment]
Irei olhar, porem funcionava normalmente!

**Comment by AnibalDuarte** [comment]
Pois é, eu vi a implementação e achei sensacional, mas o campo de busca não tem efeito.

Pergunt 1: para alterar o título da aba tem de se alterar a pasta system, correto?

Pergunta 2: seria possível customizar a moeda de venda desse leilão?

Aníbal Duarte
(51) 9 9293-7660 - Vivo| WhatsApp | Telegram | Skype
(51) 9 8312-0058 - TIM

De: Luiz Nascimento <notifications@github.com>
Enviada em: segunda-feira, 11 de fevereiro de 2019 10:09
Para: danielbarion/JBlueHeart-Source <JBlueHeart-Source@noreply.github.com>
Cc: Anibal Duarte <anibal.duarte@outlook.com>; Author <author@noreply.github.com>
Assunto: Re: [danielbarion/JBlueHeart-Source] Busca do Leilão não funciona (#14)


Irei olhar, porem funcionava normalmente!

**Comment by LuizHFallen** [comment]
Sim, na System se altera o nome das Abas do Community. Sobre customizar irei, olhar juntamente com a busca!
